tests/spaces/test_graph.py

```diff
@@ -105,14 +105,14 @@ def run_tests_with_adj(A, L, tol=1e-14, tol_m=1e-7):
     ##############################################
     # Kernel init checks
 
-    K_cons = MaternKarhunenLoeveKernel(graph, n)
+    K_cons = MaternKarhunenLoeveKernel(graph, n, normalize=False)
     params, state = K_cons.init_params_and_state()
     idx = B.cast(B.dtype(A), np.arange(n)[:, None])
 
     nu = B.cast(B.dtype(A), np.array([1.0]))
     lscale = B.cast(B.dtype(A), np.array([1.0]))
 
-    K_normed_cons = MaternKarhunenLoeveKernel(normed_graph, n)
+    K_normed_cons = MaternKarhunenLoeveKernel(normed_graph, n, normalize=False)
```

Comment:
The "normed" in `K_normed_cons` (that refers to the normalized Laplacian) conflicts somewhat with the `normalize` that refers to the normalization constant. Maybe use something like `K_nlap_cons` instead?